makefile

```diff
@@ -225,34 +241,48 @@ ASM_FLAGS += ${COMMON_FLAGS}
 
 all: $(APP_OUT_PATH)/$(PROJECT).bin # $(APP_OUT_PATH)/$(PROJECT).hex size
 
-
-.s.o:
+$(OBJ_PATH)/%.o: ../%.s
 	+@$(call MAKE_DIR,$(dir $@))
 	+@echo "Assemble: $(notdir $<)"
-	@$(AS) -c $(ASM_FLAGS) -o $@ $<
+	@$(AS) -c $(ASM_FLAGS) $< -o $@
```

Comment:
Whats the purpose of the reorder? I've forgotten the syntax...

Reply:
Reorder doesn't do anything. It was just written one way in some parts of the makefile and the other way in other parts. I'm a little OCD about consistency lol

Reply:
$@(automatic output arg) is the value for the -o flag. $<(automatic input arg) is first argument for the function